### ISSUE TITLE
empty strings are not null

### ISSUE THREAD

**Issue by SingingBush** [buggins/ddbc#34]
my app hooks into an existing schema. there are some NON NULL fields in my user table that I don't care about so I just give them an empty string. eg:

```D
    User user = new User;
    user.username = username;
    user.pass = passwdHash;
    user.nicename = "";
    user.displayname = username;
    user.email = email;
    user.url = "";
    import std.datetime;
    SysTime now = Clock.currTime(UTC());
    user.registered = cast(DateTime) now;
    session.save(user);
```

this should be fine (and is when using sqlite) but ddbc falls over when I'm using the MySQL driver:

```
/ddbc-0.2.26/ddbc/source/ddbc/drivers/mysqlddbc.d(528): MySQL error: Column 'user_nicename' cannot be null
```

**Comment by buggins** [comment]
I hope it's fixed in v0.2.29
Please confirm

**Comment by SingingBush** [comment]
This has indeed been fixed. thanks
